$NetBSD: patch-dd,v 1.4 2010/10/27 13:24:09 tsutsui Exp $

Install man pages into ${PREFIX}/man, not ${PREFIX}/share/man.

Handle arm and armeb case on NetBSD.
XXX: it looks mips is always treated as big endian.

--- configure.orig	2010-07-28 01:55:42.000000000 +0000
+++ configure	2010-08-09 20:36:55.000000000 +0000
@@ -199,13 +199,19 @@
   else
     cpu="s390"
   fi
+elif check_define __arm__ ; then
+  if check_define __ARMEL__ ; then
+    cpu="arm"
+  elif check_define __ARMEB__ ; then
+    cpu="armeb"
+  fi
 else
   cpu=`uname -m`
 fi
 
 target_list=""
 case "$cpu" in
-  alpha|cris|ia64|m68k|microblaze|ppc|ppc64|sparc64)
+  alpha|arm|armeb|cris|ia64|m68k|microblaze|ppc|ppc64|sparc64)
     cpu="$cpu"
   ;;
   i386|i486|i586|i686|i86pc|BePC)
@@ -285,7 +291,7 @@
 mingw32="no"
 EXESUF=""
 prefix="/usr/local"
-mandir="\${prefix}/share/man"
+mandir="\${prefix}/man"
 datadir="\${prefix}/share/qemu"
 docdir="\${prefix}/share/doc/qemu"
 bindir="\${prefix}/bin"
@@ -1036,7 +1042,7 @@
 
 # if cross compiling, cannot launch a program, so make a static guess
 case "$cpu" in
-  armv4b|hppa|m68k|mips|mips64|ppc|ppc64|s390|s390x|sparc|sparc64)
+  armeb|armv4b|hppa|m68k|mips|mips64|ppc|ppc64|s390|s390x|sparc|sparc64)
     bigendian=yes
   ;;
 esac
@@ -2212,7 +2218,7 @@
   i386|x86_64|alpha|cris|hppa|ia64|m68k|microblaze|mips|mips64|ppc|ppc64|s390|s390x|sparc|sparc64)
     ARCH=$cpu
   ;;
-  armv4b|armv4l)
+  arm|armeb|armv4b|armv4l)
     ARCH=arm
   ;;
 esac
